lens/interface.go

```diff
@@ -58,6 +58,7 @@ type ChainAPI interface {
 
 type StateAPI interface {
 	StateGetActor(ctx context.Context, addr address.Address, tsk types.TipSetKey) (*types.Actor, error)
+	StateLookupRobustAddress(ctx context.Context, addr address.Address, tsk types.TipSetKey) (address.Address, error)
```

Comment:
This Lotus API will use the `ForEach` to iterate address map. Maybe we could implement a new map: key is actor_id, value is robust address.